doc/dev/kernel-client-troubleshooting.rst

```diff
@@ -1,21 +0,0 @@
-====================================
```

Comment:
@joscollin why did we delete this file? Its contents were not moved elsewhere and it's useful information. I feel I must have missed this somehow during the review.

Reply:
@batrick 

This is dropped as discussed here:
https://github.com/ceph/ceph/pull/28900#discussion_r300932821.

I've checked [troubleshooting.rst](https://github.com/joscollin/ceph/blob/4d5d6272b3d3043c3f0b4b7863c89230d8c49c08/doc/cephfs/troubleshooting.rst) once again and the instructions in [kernel-client-troubleshooting.rst](https://github.com/ceph/ceph/blob/20aa4ac783e1f564019c4f8fcbe3c1f6ad4a61a9/doc/dev/kernel-client-troubleshooting.rst) are incorporated to [troubleshooting.rst](https://github.com/joscollin/ceph/blob/4d5d6272b3d3043c3f0b4b7863c89230d8c49c08/doc/cephfs/troubleshooting.rst) as mentioned [here](https://github.com/ceph/ceph/pull/28900#discussion_r300932821).